Python/README.md

```diff
@@ -129,6 +131,11 @@
 - [Stable Marriage Problem](cp/stable_marriage_problem.py)
 - [Rain Water Harvesting](cp/rainwater_harvesting.py)
 - [House Robber](cp/house_robber.py)
+- [Anagram Checker](cp/anagram_checker.py)
```

Comment:
@chicken-biryani Remove this since `- [Anagram Checker](cp/anagram_checker.py)` is already there in the `Competitive Programming` section. Also, add `Palindrome.py` program link `Competitive Programming` section.